recipes/capytaine/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "1.0" %}
+
+package:
+  name: capytaine
+  version: {{ version }}
+
+source:
+  url: https://github.com/mancellin/capytaine/archive/v{{version}}.tar.gz
+  sha256: 072de801b50b4052277b5eb3d634734a11079b69789e5fb3bccb7363d199f637
+
+build:
+  skip: True  # [py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - capytaine = capytaine.ui.cli:main
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - llvm-openmp  # [osx]
```

Comment:
I think that should be just ``openmp`` and you also have to remove ``llvm-openmp`` from build

Reply:
It is based on this section of the doc: https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp-on-macos
Is it wrong?

Reply:
Nope, you are right! sorry
This thing changed and I was not aware of it